src/devices/Arduino/FirmataCommandSequence.cs

```diff
@@ -33,14 +33,27 @@ public class FirmataCommandSequence : IEquatable<FirmataCommandSequence>
         /// Create a new command sequence
         /// </summary>
         /// <param name="command">The first byte of the command</param>
-        internal FirmataCommandSequence(FirmataCommand command = FirmataCommand.START_SYSEX)
+        internal FirmataCommandSequence(FirmataCommand command)
         {
             _sequence = new List<byte>()
             {
                 (byte)command
             };
         }
 
+        internal FirmataCommandSequence(FirmataCommand command, int pin)
+        {
+            if (pin > 15)
```

Comment:
Are all the boards with the same limit? If I'm using an ESP32 I can get more. So did I miss here?

Reply:
This is a limit of the command, not of the protocol or the board. There are some commands that have a one-byte shorthand form, but this form is limited to 15 pins, because the pin number is encoded in a nibble only. All such commands have a longer form as well, see the destinction below.